transbank/webpay/webpayplus/transaction_commit_response.py

```diff
@@ -0,0 +1,129 @@
+class TransactionCommitResponse:
+    def __init__(self, json_data):
```

Comment:
On `TransactionCreateResponse` below you used a separated `from_json()` method, but not here. Why the difference?